src/objects/core/tests/files/objecttypes_idempotent.yaml

```diff
@@ -0,0 +1,10 @@
+objecttypes_config_enable: true
```

Comment:
Do we need separate files for these tests? Most of them seem the same

Reply:
Removed the idempotent test file, the other files are quite different from each other